lib/mix/tasks/phx.gen.html.ex

```diff
@@ -190,7 +190,7 @@ defmodule Mix.Tasks.Phx.Gen.Html do
       {key, {:enum, _}}  ->
         {label(key), ~s|<%= select f, #{inspect(key)}, Ecto.Enum.values(#{inspect(schema.module)}, #{inspect(key)}), prompt: "Choose a value" %>|, error(key)}
       {key, _}  ->
-        {label(key), ~s(<%= text_input f, #{inspect(key)} %>), error(key)}
+        {label(key), ~s|<%= #{maybe_password("text_input", schema, key)} f, #{inspect(key)} %>|, error(key)}
```

Comment:
I don't think we should treat all redacts as passwords. Due to GDPR, you may want to redact SSN, emails and even addresses!

Reply:
I agree that there will be plenty of real world cases where password fields aren't needed. As this is a generator, I like the idea of giving them the password field by default, and they can swap it back if they don't want it (default secure). If we went the other way, they might not know of or think to use the password field.

Reply:
The thing is that you never want to persist a password field on the database (but rather its hash). And because this field is persisted on the database, I am afraid we can rather point people to the wrong direction by doing this.

Reply:
In my mind, the only way it would be ok to be a password would be if it is virtual and redacted but that feels a very corner case scenario that I am not sure if we should bother?

Reply:
Very good point about pointing folks in the wrong direction. I like the convenience of redact making a password field but José is right that it requires more than simply mapping a field back to the DB, so I think we need to drop the password approach as it's going to necessarily require more in user-land to be done properly